iron-autogrow-textarea.html

```diff
@@ -291,7 +291,7 @@
       // generated from the _onInput handler, and the two values are already
       // the same.
       if (textarea.value !== this.bindValue) {
-        textarea.value = this.bindValue;
+        textarea.value = !(this.bindValue || this.bindValue === 0) ? '' : this.bindValue;
```

Comment:
Exactly, `0` will be displayed as `0`, and not ''. It's hitting the `(this.bindValue == 0)` branch (note that the default branch is the negation of that), so it will use `this.bindValue`